Resync WebRTC WPT tests

youennf opened the pull request:
#### 9afd325901ac8ab9458cea914aa821172a36f1ed
<pre>
Resync WebRTC WPT tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=243334">https://bugs.webkit.org/show_bug.cgi?id=243334</a>

Reviewed by Chris Dumez.

Resynced WebRTC tests up to c0883fd.

* LayoutTests/imported/w3c/web-platform-tests/webrtc/META.yml:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCConfiguration-iceServers-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCConfiguration-iceServers.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCConfiguration-iceTransportPolicy-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCConfiguration-iceTransportPolicy.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCDataChannel-close.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCIceTransport-extension.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCIceTransport-extension.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-addIceCandidate-connectionSetup.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-addTrack.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-addTrack.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-connectionState-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-connectionState.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-createAnswer-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-createAnswer.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-createOffer-offerToReceive-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-createOffer-offerToReceive.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-getDefaultIceServers-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-getDefaultIceServers.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-getIdentityAssertion.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-getIdentityAssertion.sub.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-getStats.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-iceConnectionState-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-iceConnectionState.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-ondatachannel.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-peerIdentity-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-peerIdentity.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-relay-canvas.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-restartIce.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-restartIce.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-setRemoteDescription-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-setRemoteDescription.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-transceivers.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-transceivers.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-transport-stats.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-transport-stats.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpParameters-degradationPreference-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpParameters-degradationPreference.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpReceiver-getStats.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpReceiver-getStats.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver-setDirection-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver-setDirection.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver-stopping.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver-stopping.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCTrackEvent-fire-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RollbackEvents.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RollbackEvents.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/identity-helper.sub.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/idlharness.https.window.js:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/README.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCPeerConnection-addStream.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCPeerConnection-addStream.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCPeerConnection-createOffer-offerToReceive-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCPeerConnection-createOffer-offerToReceive.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCRtpTransceiver-with-OfferToReceive-options.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCRtpTransceiver-with-OfferToReceive-options.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/onaddstream.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/onaddstream.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/bundle.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/bundle.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/dtls-setup.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/dtls-setup.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/h264-profile-levels.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/h264-profile-levels.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtp-demuxing.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtp-extension-support-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtp-extension-support.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtp-payloadtypes-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtp-payloadtypes.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtx-codecs.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtx-codecs.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/simulcast-answer-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/simulcast-answer.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/receiver-track-live.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/recvonly-transceiver-can-become-sendrecv.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/recvonly-transceiver-can-become-sendrecv.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/basic.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/basic.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/getStats.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/getStats.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/h264.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/h264.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/setParameters-active.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/setParameters-active.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/simulcast.js:
(async negotiateSimulcastAndWaitForVideo):
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/vp8.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/vp8.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/w3c-import.log:
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/253513@main">https://commits.webkit.org/253513@main</a>
</pre>

cdumez reviewed (approved):
LGTM. Would be nice to specify the git revision in the commit log.

youennf committed:
Resync WebRTC WPT tests
https://bugs.webkit.org/show_bug.cgi?id=243334

Reviewed by Chris Dumez.

Resynced WebRTC tests up to c0883fd.

* LayoutTests/imported/w3c/web-platform-tests/webrtc/META.yml:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCConfiguration-iceServers-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCConfiguration-iceServers.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCConfiguration-iceTransportPolicy-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCConfiguration-iceTransportPolicy.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCDataChannel-close.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCIceTransport-extension.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCIceTransport-extension.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-addIceCandidate-connectionSetup.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-addTrack.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-addTrack.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-connectionState-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-connectionState.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-createAnswer-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-createAnswer.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-createOffer-offerToReceive-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-createOffer-offerToReceive.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-getDefaultIceServers-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-getDefaultIceServers.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-getIdentityAssertion.sub-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-getIdentityAssertion.sub.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-getStats.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-iceConnectionState-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-iceConnectionState.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-ondatachannel.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-peerIdentity-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-peerIdentity.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-relay-canvas.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-restartIce.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-restartIce.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-setRemoteDescription-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-setRemoteDescription.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-transceivers.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-transceivers.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-transport-stats.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-transport-stats.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpParameters-degradationPreference-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpParameters-degradationPreference.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpReceiver-getStats.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpReceiver-getStats.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver-setDirection-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver-setDirection.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver-stopping.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver-stopping.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCRtpTransceiver.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCTrackEvent-fire-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RollbackEvents.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/RollbackEvents.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/identity-helper.sub.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/idlharness.https.window.js:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/README.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCPeerConnection-addStream.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCPeerConnection-addStream.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCPeerConnection-createOffer-offerToReceive-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCPeerConnection-createOffer-offerToReceive.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCRtpTransceiver-with-OfferToReceive-options.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/RTCRtpTransceiver-with-OfferToReceive-options.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/onaddstream.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/onaddstream.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/legacy/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/bundle.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/bundle.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/dtls-setup.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/dtls-setup.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/h264-profile-levels.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/h264-profile-levels.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtp-demuxing.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtp-extension-support-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtp-extension-support.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtp-payloadtypes-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtp-payloadtypes.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtx-codecs.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/rtx-codecs.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/simulcast-answer-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/simulcast-answer.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/protocol/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/receiver-track-live.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/recvonly-transceiver-can-become-sendrecv.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/recvonly-transceiver-can-become-sendrecv.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/basic.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/basic.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/getStats.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/getStats.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/h264.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/h264.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/setParameters-active.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/setParameters-active.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/simulcast.js:
(async negotiateSimulcastAndWaitForVideo):
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/vp8.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/simulcast/vp8.https.html:
* LayoutTests/imported/w3c/web-platform-tests/webrtc/w3c-import.log:
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/253513@main
